cetmix_tower_server/README.rst

```diff
@@ -57,94 +57,96 @@ Changelog
 16.0.1.1.4 (2025-07-07)
 -----------------------
 
-- Bugfixes: Command log sorting (4816)
+-  Bugfixes: Command log sorting (4816)
```

Comment:
This file should not be changed, could you please check if you have the latest version of the main branch locally and run the pre-commit again?